Samples/PowerShell/HA.PoolOperations.ps1

```diff
@@ -0,0 +1,54 @@
+# This sample script demonstrates the use of verbs provided for manipulating with High Availability pools.
+#
+# The script progresses in several steps:
+# 1. Selecting a pool by name.
+# 2. Executing "Manual Failover" on the pool
+# 3. Executing "Disable Pool" on the pool
+#
+# Please update the hostname and credential setup to match your configuration.
+# Please note that this scripts can affect your pools!!!
+
+if (-not (Get-PSSnapin | where {$_.Name -eq "SwisSnapin"})) {
+    Add-PSSnapin "SwisSnapin"
+}
+
+# Connect to SWIS
+$hostname = "localhost"
+$username = "admin"
+$password = New-Object System.Security.SecureString
+$cred = New-Object -typename System.Management.Automation.PSCredential -argumentlist $username, $password
```

Comment:
There's no need to use this awkward `PSCredential` thing any more. Just use the `-Username` and `-Password` options for `Connect-Swis`.